win_installer.py

```diff
@@ -187,12 +190,21 @@ def create_pynsist_cfg(python, repo_root, filename, encoding="latin1"):
     icon_file = os.path.join(repo_root, "package", "icons", "win_icon.ico")
     license_file = os.path.join(repo_root, "LICENSE")
 
+    # On Linux Debian systems "pkg-resources" is erroneously reported.
+    # This is a bug in Debian's patched version of pip.
+    excluded_packages = {mu_package_name, "pkg-resources", "noise"}
```

Comment:
Understand the note on `pkg-resources`. What's the case with `noise`? Do we need to extend the comment a bit or should `noise` not be in the `excluded_packages` set?